Plugins/Creature/Creature.hpp

```diff
@@ -83,6 +83,10 @@ class Creature : public NWNXLib::Plugin
     ArgumentStack LevelDown                     (ArgumentStack&& args);
     ArgumentStack SetChallengeRating            (ArgumentStack&& args);
 
```

Comment:
Oh, and while you're resolving the merge, please humor my OCD and remove this newline.

Reply:
Lol.  Can do